src/main/kotlin/step3/OilStation.kt

```diff
@@ -0,0 +1,9 @@
+package step3
+
+object OilStation {
+    private val randomRange = (0..9)
```

Comment:
상수 표기법을 따라주는 게 좋겠네요~
```suggestion
    private val RANDOM_RANGE = (0..9)
```

Reply:
확실히 상수 표기법 따르는 편이 구분 잘되고 좋네요~👍